### DISCUSSION TITLE
Build Error while using sassOptions fatalDeprecations 'mixed-decls' in modern-compiler

### DISCUSSION THREAD

**Discussion by raffi-46** [webpack/webpack#18775] [Q&A]
![image](https://github.com/user-attachments/assets/2454f744-4579-49e2-ada0-21fb622434ba)

![image](https://github.com/user-attachments/assets/d60df653-2114-4061-9cb4-36f92e627e13)

![image](https://github.com/user-attachments/assets/9318c75d-17ca-449a-bded-39b435b15130)

**Comment by raffi-46** [comment]
@alexander-akait Looks all perfect in app but eventhough no mixed-decls still unwanted error logging

**Reply by alexander-akait** [reply]
@raffi-46 Can you create reprducible test repo?

**Reply by raffi-46** [reply]
ok i will share like a string "mixed-decls" it must be a array of strings ['mixed-decls'] ? I hope Syntax correct ? @alexander-akait

**Reply by alexander-akait** [reply]
https://sass-lang.com/documentation/js-api/interfaces/options/#fatalDeprecations, yes it should be an array
